public/static/docs/command-reference/remote/modify.md

```diff
@@ -403,6 +403,69 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
 
 </details>
 
+<details>
+
+### Click for HTTP
+
+- `auth` - authentication method to use when accessing a remote. Accepted values
+  are:
+
+  - `basic` - use HTTP basic access authentication (`user` and `password`
+    options should also be provided).
+  - `digest` - use HTTP digest access authentication (`user` and `password`
+    options should also be provided).
+  - `custom` - use a custom HTTP header field and value when accessing a remote
+    (`custom_auth_header` and `password` options should also be provided). When
+    `auth` is set to `custom`, an additional HTTP header field in the form of
+    `custom_auth_header: password` will be set for all HTTP requests to a
+    remote.
```

Comment:
``An additional HTTP header field will be set for all HTTP requests to the remote in the form: `custom_auth_header: password`. `custom_auth_header` and `password` (or `ask_password`) parameters should also be configured. ``

Not sure about "(or `ask_password`)" here.

Reply:
`ask_password` can technically still be used to set the password/header field value here, so I'll go ahead and document it

Reply:
Thanks!